src/pages/docs/administration/migrate-spaces-with-octoterra/index.md

```diff
@@ -0,0 +1,309 @@
+:::
+
+### Migrating projects
+
+Here is the command to export a project to a Terraform module for Linux and macOS:
+
+```Bash
+./octoterra \
+  -url https://yourinstance.octopus.app \
+  -space Spaces-1234 \
+  -apiKey API-XXXXXXXXXXXXXXXXXXX \
+  -projectName "My Project" \
+  -lookupProjectDependencies \
+  -dest ~/Desktop/project
+```
+
+Here is the command for Windows and Powershell:
+
+```Powershell
+./octoterra `
+  -url https://yourinstance.octopus.app `
+  -space Spaces-1234 `
+  -apiKey API-XXXXXXXXXXXXXXXXXXX `
+  -projectName "My Project" `
+  -lookupProjectDependencies `
+  -dest ~/Desktop/project
+```
+
+:::div{.hint}
+If your project or runbooks include step templates and the project will be recreated in a new space, you must detach the step templates using the `-detachProjectTemplates` argument:
+
+```Bash
+./octoterra \
+  -url https://yourinstance.octopus.app \
+  -space Spaces-1234 \
+  -apiKey API-XXXXXXXXXXXXXXXXXXX \
+  -projectName "My Project" \
+  -lookupProjectDependencies \
+  -detachProjectTemplates \
+  -dummySecretVariableValues \
+  -dest ~/Desktop/project
+```
+
+When the project is recreated, the step template steps will be recreated in a detached state.
+
+If you get an error like this when applying a module, it can mean that a project or runbook is referencing a step template that does not exist in the target space, and you must use the `-detachProjectTemplates` argument:
+
+```Bash
+Error: Octopus API error: Resource is not found or it doesn't exist in the current space context. Please contact your administrator for more information. [] 
+```
+:::
+
+In this example, the Terraform module is created in the directory `~/Desktop/project/space_population`.
+
+To recreate the project, enter the directory:
+
+```Bash
+cd ~/Desktop/project/space_population
+```
+
+Initialize the Terraform module:
+
+```Bash
+terraform init
+```
+
+Then apply the module:
+
+```Bash
+terraform apply
+```
+
+Terraform will prompt you for the Octopus server URL, API Key, space ID, and for the value of any sensitive variables.
+
+:::div{.hint}
+The `-dummySecretVariableValues` option with can also be used when exporting projects to place a dummy value into any exported sensitive variables:
+
+```Bash
+./octoterra \
+  -url https://yourinstance.octopus.app \
+  -space Spaces-1234 \
+  -apiKey API-XXXXXXXXXXXXXXXXXXX \
+  -projectName "My Project" \
+  -lookupProjectDependencies \
+  -detachProjectTemplates \
+  -dummySecretVariableValues \
+  -dest ~/Desktop/project
+```
+
+The values of sensitive variables must be manually updated after the module is applied.
+:::
+
+## Recreating Config-as-Code projects
+
+CaC projects can not share a Git repository. When a CaC project is recreated, it must point to a new Git repository. This is done by overriding the default value of a Terraform variable in the exported module.
```

Comment:
It might be handy to mention that if they're migrating from a self-hosted instance to Octopus Cloud that this doesn't apply.